download_models.sh

```diff
@@ -0,0 +1,4 @@
+curl -o models/es-ca.zip https://object.pouta.csc.fi/OPUS-MT-models/es-ca/opus-2020-01-16.zip
```

Comment:
Just a suggestion: maybe it would be better to use this one: https://object.pouta.csc.fi/Tatoeba-MT-models/spa-cat/opus-2021-02-19.zip
I think it is the same one (claims same BLEU) but it is the newest version I founded.